.github/workflows/build.yml

```
@@ -0,0 +1,28 @@
+name: Build GWT Site
+
+on:
+  push:
+#    branches: [ 'main' ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Java 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'temurin'
+          cache: 'maven'
+      - name: Build with Maven
+        run: mvn verify -Pgrunt -e --batch-mode
```

Add `--no-transfer-progress` (or `-ntp`) to make the logs readable

```suggestion
        run: mvn verify -Pgrunt -e --batch-mode --no-transfer-progress
```